docs/src/components/pricing/PricingTable.tsx

```diff
@@ -869,6 +879,9 @@ const proData: Record<string, React.ReactNode> = {
   'charts/filter-interaction': pending,
   'charts/selection-interaction': no,
 
+  'tree-view/simple-tree-view': yes,
+  'tree-view/rich-tree-view': yes,
```

Comment:
@joserodolfofreitas we can prepare a follow up with upcoming tree view pro features